components/ResultsPage/BenefitCard.tsx

```diff
@@ -79,17 +83,22 @@ export const BenefitCard: React.VFC<{
 
       <div className="mt-4">
         {links &&
-          links.map(({ text, url, icon, alt }, index) => (
+          links.map(({ text, url, icon, alt, action }, index) => (
             <div key={index} className="flex items-center py-4 text-content">
               <div>
                 <Image src={`/${icon}.png`} alt={alt} width="30" height="44" />
               </div>
               <div className="pl-5 w-full">
-                <DSLink
-                  id={`${benefitKey}Link${index}`}
-                  href={url}
-                  text={text}
-                />
+                <span
+                  data-gc-analytics-customclick={`${AA_BUTTON_CLICK_ATTRIBUTE}:${action}`}
+                  onClick={() => console.log('click link')}
```

Comment:
Did you mean to leave in the console log?

Reply:
NO! just removed it. Nice catch